test/ParseableInterface/Conformances.swiftinterface

```diff
@@ -11,6 +11,22 @@ public protocol MyProto {
   var prop: Int { get set }
   subscript(index: Int) -> Int { get set }
 }
+extension MyProto {
+  public func method() {}
+}
+
+// Make sure the default witness table is empty.
```

Comment:
Can you change SILGen to not emit one at all?

Reply:
Sure, that's also a good idea.